src/v2/user_account.rs

```diff
@@ -0,0 +1,214 @@
+use std::time::SystemTime;
+
+use serde::{Serialize, Deserialize};
+
+use ruma::api::ruma_api;
+use ruma::identifiers::UserId;
+use ruma::api::client::r0::contact::get_contacts::ThirdPartyIdentifier;
+
+use ruma::api::client;
```

Comment:
Please merge the ruma imports into one and remove the newline between the serde and ruma imports (c.f. https://github.com/ruma/ruma/blob/main/CONTRIBUTING.md#import-formatting).